packages/salesforcedx-apex-debugger/src/commands/requestService.ts

```diff
@@ -80,15 +80,28 @@ export class RequestService {
       command.getQueryString() == null
         ? urlElements.join('/')
         : urlElements.join('/').concat('?', command.getQueryString()!);
-    const options: XHROptions = {
-      type: 'POST',
-      url: requestUrl,
-      headers: {
-        'Content-Type': 'application/json',
-        Accept: 'application/json',
-        Authorization: `OAuth ${this.accessToken}`
-      }
-    };
+    const options: XHROptions =
+      command.getRequest() == null
+        ? {
+            type: 'POST',
+            url: requestUrl,
+            headers: {
+              'Content-Type': 'application/json',
+              Accept: 'application/json',
+              Authorization: `OAuth ${this.accessToken}`
+            }
+          }
+        : {
+            type: 'POST',
+            url: requestUrl,
+            headers: {
+              'Content-Type': 'application/json',
+              Accept: 'application/json',
+              Authorization: `OAuth ${this.accessToken}`
+            },
+            data: JSON.stringify(command.getRequest())
```

Comment:
Could we use if-else shorthand here?

Reply:
I started from that. However, that puts an empty "data" into the options which would require rework of all existing tests. Not sure if it's worth it.